packages/core/src/modules/indy/repository/AnonCredsCredentialDefinitionRecord.ts

```diff
@@ -1,39 +1,47 @@
+import type { DidIndyNamespace } from '../../../utils/indyIdentifiers'
 import type { CredDef } from 'indy-sdk'
 
 import { BaseRecord } from '../../../storage/BaseRecord'
-import { didFromCredentialDefinitionId } from '../../../utils/did'
+import { getQualifiedIdentifier } from '../../../utils/indyIdentifiers'
+import { CredentialDefinitionTransformer } from '../../../utils/transformers'
+import { uuid } from '../../../utils/uuid'
 
 export interface AnonCredsCredentialDefinitionRecordProps {
   credentialDefinition: CredDef
+  didIndyNamespace: DidIndyNamespace
```

Comment:
I think this isn't needed

Reply:
let me double-check this. Added it for the transformer.